File: Example/PDStrategy/PDStrategy/PDStrategy/Views/PDValueLabel.m
//
//  PDValueLabel.m
//  PDStrategy
//
//  Created by Pavel Deminov on 19/11/2017.
//  Copyright © 2017 Pavel Deminov. All rights reserved.
//

#import "PDValueLabel.h"

@implementation PDValueLabel

- (void)setup {
    [super setup];
    self.font = [UIFont systemFontOfSize:14];
    self.textColor = [UIColor darkGrayColor];
    self.numberOfLines = 0;
}

@end

File: Example/PDStrategy/PDStrategy/PDStrategy/Views/PDValueLabel.swift
//
//  PDValueLabel.swift
//  PDStrategy
//
//  Created by Pavel Deminov on 05/11/2017.
//  Copyright © 2017 Pavel Deminov. All rights reserved.
//

import UIKit

class PDValueLabel: PDLabel {
    
    override func setup() {
        super.setup()
        
        font = UIFont.systemFont(ofSize: 14)
        textColor = UIColor.darkGray
        numberOfLines = 0
    }

}
